README.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Restaurant App
+
+It's a Restaurant app to show the restaurant, the menu and to contact the owner information dynamically by clicking in the button in the Nav Bar
+
+![screenshot](./assets/screenshot/screenshot.jpg)
+
+- Has the information about the restaurant
+- Shows the menu.
+- Has a contact section.
+
+## Built With
+
+- Html, CSS, and JavaScript
+- Bootstrap
+
+## Live Demo
+
+[Live Demo Link](https://fdi96.github.io/Restaurant-App/)
```

Comment:
- [ ] [MANDATORY] This live-demo link is returning page not found. please, make sure the live-demo link is valid.

Reply:
Done! Thanks for the review.